### PR TITLE
including custom protos resolution

### LINKED ISSUE #36: Protobuf Extensions Support
This is more of a question rather than issue.  Is it planned or even possible to support protobuf extensions?

### PR DESCRIPTION
**Pull request by fenos**
This PR fixes #36 and #30 

Introduces a way of adding custom proto files which will be automatically resolved during an import

the list includes:

- google/protobuf/api.proto
- google/protobuf/descriptior.proto
- google/protobuf/source_context.proto
- google/protobuf/type.proto

- google/rpc/status.proto

relates to #51 

For now the protos will be included directly within the project, probably in the future i'd like to create a build script to copy them from `node_modules`

### CODE REVIEW

File: webpack.config.renderer.dev.js [hunk 1]
```diff
@@ -192,7 +192,10 @@ module.exports = merge.smart(baseConfig, {
 
     new webpack.LoaderOptionsPlugin({
       debug: true
-    })
+    }),
+    new webpack.DefinePlugin({
+      __static: `"${path.join(configurator.projectDir, "static").replace(/\\/g, "\\\\")}"`,
```
Comment (bjaglin) [inline webpack.config.renderer.dev.js]: I can't seem to run this in dev-mode after a clean `git clone` & `yarn install`:
```
$ npm run start-server-dev

> bloomrpc@1.2.1 start-server-dev /home/bjaglin/git/projects/bloomrpc
> cross-env NODE_ENV=development node --trace-warnings ./node_modules/webpack-dev-server/bin/webpack-dev-server --config webpack.config.renderer.dev.js

/home/bjaglin/git/projects/bloomrpc/webpack.config.renderer.dev.js:197
      __static: `"${path.join(configurator.projectDir, "static").replace(/\\/g, "\\\\")}"`,
                              ^

ReferenceError: configurator is not defined
    at Object.<anonymous> (/home/bjaglin/git/projects/bloomrpc/webpack.config.renderer.dev.js:197:31)
    at Module._compile (internal/modules/cjs/loader.js:799:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:810:10)
    at Module.load (internal/modules/cjs/loader.js:666:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:606:12)
    at Function.Module._load (internal/modules/cjs/loader.js:598:3)
    at Module.require (internal/modules/cjs/loader.js:705:19)
    at require (internal/modules/cjs/helpers.js:14:16)
    at WEBPACK_OPTIONS (/home/bjaglin/git/projects/bloomrpc/node_modules/webpack-cli/bin/convert-argv.js:115:13)
    at requireConfig (/home/bjaglin/git/projects/bloomrpc/node_modules/webpack-cli/bin/convert-argv.js:117:6)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
```

Comment (fenos) [reply webpack.config.renderer.dev.js]: Good catch! i forgot to replace that variable

Comment (fenos) [reply webpack.config.renderer.dev.js]: @bjaglin it's now fixed

**Comment by fenos** [comment]
Super 🎉

**Comment by eric-taix-teads** [comment]
Great job 👍 Thanks

**Comment by risentveber** [comment]
I would like some kind of setting that will provide include path for any referenced proto file. Because of hard-coded definitions seem like temporary solution

**Pull request merged**